railties/test/generators/app_generator_test.rb

```diff
@@ -56,6 +56,13 @@ class AppGeneratorTest < Rails::Generators::TestCase
   # brings setup, teardown, and some tests
   include SharedGeneratorTests
 
+  def assert_generates_with_bundler(options = {})
+    generator([destination_root], options)
+    assert_called_with(generator, :bundle_command, [ ['install'], ["exec spring binstub --all"] ]) do
+      quietly { generator.invoke_all }
+    end
```

Comment:
@kaspth Lets say we put this expectation 
````ruby
mock_generator.expect(:call, nil) {|arg| arg == 'install' }
mock_generator.expect(:call, nil) {|arg| arg == 'exec spring binstub --all' }
````
We still have to call bundle_command two times. but second expectation is optional it will only be fulfilled if spring is enabled.

So can we do something like this
````ruby
mock_generator.expect(:call, nil) {|arg| arg == 'install' }
mock_generator.expect(:call, nil) {|arg| arg == 'exec spring binstub --all' } if generator.send(:spring_install?)
````